include/fast_double_parser.h

```diff
@@ -1110,9 +1110,13 @@ static const char * parse_float_strtod(const char *ptr, double *outDouble) {
 #elif defined(_WIN32)
   static _locale_t c_locale = _create_locale(LC_ALL, "C");
   *outDouble = _strtod_l(ptr, &endptr, c_locale);
+#else
```

Comment:
Could this be done with 
```
#elif
```

Reply:
Thanks for your suggestions. This looks better. :-). Pushed the new code. thanks.